cmd/main.go

```diff
@@ -63,8 +64,11 @@ func handle() {
 
 	gceDriver := driver.GetGCEDriver()
 
-	//Initialize GCE Driver (Move setup to main?)
-	cloudProvider, err := gce.CreateCloudProvider(vendorVersion, *gceConfigFilePath)
+	//Initialize GCE Driver
+	ctx, cancel := context.WithCancel(context.Background())
```

Comment:
I'm not sure if this will solve the issue? With this change, will we end up canceling all outstanding operations whenever the driver restarts?

Have you tried to repro the issue by setting a short timeout?

Reply:
unliked the issue, this PR doesn't have anything to do with it other than the fact they both involve contexts :)